src/select.coffee

```diff
@@ -354,14 +386,23 @@ Ember.AddeparMixins.ResizeHandlerMixin,
   deletePressed: Ember.K
 
   escapePressed: (event) ->
-    @send 'hideDropdown'
+    if @get('showDropdown')
+      @send 'hideDropdown'
+      @$()?.focus()
```

Comment:
It is somehow null in the scenario where we put it on a carousel popover, and after hitting Enter, the carousel select disappears, and @$() is null. So I just wanted to make sure it not happening for ESC too